src/invariant/test/BucketListIsConsistentWithDatabaseTests.cpp

```diff
@@ -416,6 +416,21 @@ class ApplyBucketsWorkModifyEntry : public ApplyBucketsWork
         entry.data.data().dataName = data.dataName;
     }
 
+    void
+    modifyClaimableBalanceEntry(LedgerEntry& entry)
```

Comment:
"BucketListIsConsistentWithDatabase modified entries" doesn't work. I think you got this backwards: it should say `entry.data.claimableBalance().balanceID = cb.balanceID` on line 426, and 427-431 should be deleted (the point is to modify the entry but keep the same key).

Reply:
Fixed. Didn't realize a lot of these were acceptance tests.